### PR TITLE
feat: issue #17/ focus on next player name input when pressing enter

### PR DESCRIPTION
**Pull request by Viou**
feat: issue #17/ focus on next player name input when pressing enter

### CODE REVIEW

**Comment by Viou** [comment]
The Input FC wasn't playing nicely with React ref (error: Function components cannot be given refs. Attempts to access this ref will fail) so I refactored to use a regular html input. My React is a bit rusty though, so if there's a strong motivation to use the Input component I can look into further refactoring.

**Comment by Kedyn** [comment]
So, after this change would we still be using Input component or can we just remove it?

**Comment by Viou** [comment]
The only other use would be in the Header component

File: src/components/Player/Player.tsx [hunk 1]
```diff
@@ -56,14 +68,17 @@ export default function Player(props: IPlayerProps): JSX.Element {
...
       </div>
       {names && (
-        <div className={classes.name}>
-          <Input
+        <div className={playerStyles.name}>
+          <input
+            type="text"
             placeholder="Player"
-            className={classes.input}
+            className={`${playerStyles.input} ${inputStyles.root}`}
```
Comment (Kedyn) [inline src/components/Player/Player.tsx:75]: There seems to be a problem with the text position, is center... I think you can just remove the inputStyles and style it using the components styles, we still do not know if the header will be removed or not.


PS: if you plan on contributing a bit more on this project, we will be having a small meeting about what we will focus on next on discord, add me if interested: SykO#9032

**Review by Kedyn — CHANGES_REQUESTED** [review]

**Review by Kedyn — APPROVED** [review]

**Pull request merged**